docs/_docs/quickstart.md

```diff
@@ -4,25 +4,97 @@ title: Quick-start guide
 permalink: /docs/quickstart/
 ---
 
-For the impatient, here's how to get a boilerplate Jekyll site up and running.
+If you already have [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements/)), you can create a new Jekyll site by doing the following:
 
 ```sh
-~ $ gem install jekyll bundler
-~ $ jekyll new myblog
+~ $ gem install jekyll bundler 
+# installs Jekyll and Bundler through RubyGems
+~ $ jekyll new myblog 
+# Creates a new Jekyll site called myblog
 ~ $ cd myblog
```

Comment:
I feel comments should be *before* the code to be run.. and followed by a clear line for readability
```sh
# install both Jekyll & Bundler from Rubygems
$ gem install jekyll bundler

# create a new Jekyll site at /myblog
$ jekyll new myblog
```
`jekyll new` *actually* takes the argument as `[PATH]` instead of `[NAME]`. Running `jekyll help new` will clarify it.. I'm only specifying because `jekyll new blog/docs` will create a site at `./blog/docs`, still titled `your awesome site`